project-base/app/src/DataFixtures/Demo/ReadyCategorySeoDataFixture.php

```diff
@@ -54,13 +54,13 @@ public function load(ObjectManager $manager)
             'ordering' => ProductListOrderingConfig::ORDER_BY_PRIORITY,
             'parameterValueIdsByParameterIds' => [
                 // 'Water Tank Volume' => 2 l
-                38 => 75,
+                38 => $this->parameterFacade->getParameterValueByValueTextAndLocale(t('2 l', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $firstDomainLocale), $firstDomainLocale)->getId(),
```

Comment:
love it! :heart_eyes: 
If we wanted to make it perfect, we could rewrite the parameters' IDs as well :innocent: However, that would require more changes (this is on my wishlist for the future) - we would need to add references for the parameter's data fixtures that are now (mostly) created in `ProductDataFxiture::findParameterByNamesOrCreateNew`, or create a separate data fixture for the parameters, or something like that :slightly_smiling_face:

Reply:
Yes